lib/operations/db_collections.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const Collection = require('../collection');
+const handleCallback = require('../utils').handleCallback;
+
+function collections(self, options, callback) {
+  options = Object.assign({}, options, { nameOnly: true });
+  // Let's get the collection names
+  self.listCollections({}, options).toArray(function(err, documents) {
```

Comment:
`toArray(function(err, documents) {` can be converted to use big arrow notation: `toArray((err, documents) => {`